packages/website/src/components/Modals/RecruitmentModal.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+
+import {
+  Modal,
+  ModalBody,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+} from '@coderscamp/ui/components/Modal';
+import { Typography } from '@coderscamp/ui/components/Typography';
+
+import { forMentors } from './modal.data';
+import { ModalForm } from './ModalForm';
+
+type RecruitmentModalProps = {
+  isOpen: boolean;
+  onClose: () => void;
+  modalText?: {
+    header: string;
+    body: string;
+    footer: string;
+  };
+};
+
+export const RecruitmentModal = ({ isOpen, onClose, modalText = forMentors }: RecruitmentModalProps) => {
```

Comment:
💭 It could be easier to understand if we always enforce to pass the `modalText` prop instead of making one of them the default